contrib/terraform/gcp/sample-inventory/cluster.tfvars

```diff
@@ -0,0 +1,78 @@
+###### Environment Details or ######
+env = "open-source"
+
+######### user_name #
+user_name = ""
+
+######## Region ##############
+region = "us-central1"
+
+######### GCP project name ############### 
+gcp_project_id = ""
+
+###### GCP Service Account json file folder location ##########
+gcp_service_account = "/builds/kargo-ci/kubernetes-sigs-kubespray/contrib/terraform/gcp/service-account.json"
+
+
+######### Kube automation folder where you would like to clone the kubespray project #################
+kube_automation_folder = "/home/app/kube_automation"
+
+############# ssh-key file path ############
+ssh_key = "/builds/kargo-ci/kubernetes-sigs-kubespray/contrib/terraform/gcp/ssh-key"
```

Comment:
Don't think using `~/.ssh/id_rsa` will be a good approach. Because the user who is setting up the cluster might don't want to use his private key, there might be different dedicated `user` id for these purposes. Though it depends/varies from org to org.